.github/workflows/lint.yml

```
@@ -62,4 +62,20 @@ jobs:
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@2.0.0
         with:
-          scandir: './libevm'
+          scandir: "./libevm"
+
+  markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Lint
+        uses: DavidAnson/markdownlint-cli2-action@05f32210e84442804257b2a6f20b273450ec8265 # v19.1.0
+        with:
+          globs: |
+            libevm/**.md
+          config: .github/workflows/configs/.markdownlint.json
```

```suggestion
          config: .github/workflows/configs/markdownlint.json
```

This is already in the hidden `.github` directory so no point hiding the file too.

---

Actually the only point is to have it match the file icon and json schema, given the default markdown lint config file name is `.markdownlint.json`. But I can remove the leading dot if you feel it's important